src/Common/src/Interop/Windows/sspicli/SecuritySafeHandles.cs

```diff
@@ -1133,22 +1051,15 @@ public new IntPtr DangerousGetHandle()
             ref SafeDeleteContext refContext,
             SecurityBuffer[] inSecBuffers)
         {
-            if (GlobalLog.IsEnabled)
+            if (NetEventSource.IsEnabled)
             {
-                GlobalLog.Enter("SafeDeleteContext::CompleteAuthToken");
-                GlobalLog.Print("    refContext       = " + LoggingHash.ObjectToString(refContext));
-#if TRACE_VERBOSE
-                GlobalLog.Print("    inSecBuffers[]   = length:" + inSecBuffers.Length);
-#endif
+                NetEventSource.Enter(null, "SafeDeleteContext::CompleteAuthToken");
```

Comment:
This would be better handled in a single Enter event instead of Enter plus two Infos.  (That may be beyond the scope of this change, but can we at least file general issues for obviously bad stuff like this?)